packages/sirius-web/frontend/sirius-web-application/package.json

```diff
@@ -39,16 +39,21 @@
     "@eclipse-sirius/sirius-components-deck": "*",
     "@eclipse-sirius/sirius-components-portals": "*",
     "@eclipse-sirius/sirius-components-trees": "*",
+    "@floating-ui/dom": "^1.1.1",
+    "@floating-ui/react": "^0.19.0",
     "@material-ui/core": "4.12.4",
     "@material-ui/icons": "4.11.3",
     "@material-ui/lab": "4.0.0-alpha.61",
-    "@ObeoNetwork/gantt-task-react": "0.3.13",
+    "@ObeoNetwork/gantt-task-react": "0.4.4",
```

Comment:
I don't think sirius-web-application need these dependencies, this module doesn't use them...